posthog/settings/session_replay.py

```diff
@@ -15,3 +17,9 @@
 REALTIME_SNAPSHOTS_FROM_REDIS_ATTEMPT_TIMEOUT_SECONDS = get_from_env(
     "REALTIME_SNAPSHOTS_FROM_REDIS_ATTEMPT_TIMEOUT_SECONDS", 0.2, type_cast=float
 )
+
+REPLAY_EMBEDDINGS_CALCULATION_CELERY_INTERVAL_SECONDS = get_from_env(
+    "REPLAY_EMBEDDINGS_CALCULATION_CELERY_INTERVAL_SECONDS", 300, type_cast=int
```

Comment:
I'm assuming the default is `300` - is that not a very short interval on which we would be rerunning the job?